### PR TITLE
How-to: Article how to accessible heading structure

### PR DESCRIPTION
**Pull request by rianrietveld**
Headings are the backbone of the content of a page. A visitor should be able to scan or navigate a webpage using headings to get a good impression of its content.

This PR adds a page to the posts / How-to section.

### CODE REVIEW

**Comment by ericwbailey** [comment]
This looks incredible at a cursory skim, @rianrietveld. I'll review it tonight, but I'm very excited to see it here!

While I'm reviewing, a question for you: would you like to publish it as soon as its ready, or would you prefer to wait until holidays/New Year's Eve celebrations are over?

File: src/posts/how-to-accessible-heading-structure.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+
+> It’s generally agreed that how you use headings doesn’t specifically impact your SEO. Making minor tweaks to individual headings likely won’t help your performance. However, there are indirect benefits. Using headings creates texts of higher quality that are also easier to read. A better text is better for users, which is better for your SEO.
+
+Write well-structured headings, because what is good for your reader, is also good for your SEO.
+
+## Tools
+
+Next to the automated check tools like [Axe Devtools](https://www.deque.com/axe/devtools/), there are some browser add-ons and CMS functionality to quickly check the heading structure in a visual way.
+
+### HeadingsMap by Jorge Rumoroso
```
Comment (ericwbailey) [inline src/posts/how-to-accessible-heading-structure.md:256]:
```suggestion
### HeadingsMap by Jorge Rumoroso

```

File: src/posts/how-to-accessible-heading-structure.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+The browser add-on HeadingsMap by Jorge Rumoroso gives a clear visual presentation of the heading structure and indicates errors for skipping levels and incorrect order.
+
+- [HeadingsMap for Chrome](https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi)
+- [HeadingsMap for FireFox](https://addons.mozilla.org/en-GB/firefox/addon/headingsmap/)
+- [HeadingsMap for Edge](https://microsoftedge.microsoft.com/addons/detail/headingsmap/bokekiiaddinealohkmhjcgfanndmcgo)
+
+![Screenshot of how HeadingsMap visualises the heading structure in a sidebar of the homepage of the Accessibility Project](/img/posts/how-to-accessible-heading-structure/headings-map.png)
+
+### WAVE Evaluation Tool by WebAIM
```
Comment (ericwbailey) [inline src/posts/how-to-accessible-heading-structure.md:265]:
```suggestion
### WAVE Evaluation Tool by WebAIM 

```

File: src/posts/how-to-accessible-heading-structure.md [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+### WAVE Evaluation Tool by WebAIM 
+This tools shows, embedded within the landmarks, a visual presentation of the heading structure.
+
+- Online at [wave.webaim.org](https://wave.webaim.org/)
+- [WAVE for Chrome](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
+- [WAVE for FireFox](https://addons.mozilla.org/en-GB/firefox/addon/wave-accessibility-tool/)
+
+![Screenshot of how Wave visualises the heading structure in a sidebar the homepage of the Accessibility Project](/img/posts/how-to-accessible-heading-structure/wave.png)
+
+### WordPress
```
Comment (ericwbailey) [inline src/posts/how-to-accessible-heading-structure.md:274]:
```suggestion
### WordPress 

```

**Review by ericwbailey — CHANGES_REQUESTED** [review]
@rianrietveld This is a tour de force, and I'm so excited to have it added to our site. Thank you so much for such a comprehensive and clearly-written post!

The review comment count might look a little intimidating at first, but it's mostly suggested edits for adding newlines to make our Markdown parser happy. After that there's a few questions for you.

**Comment by rianrietveld** [comment]
Hey @ericwbailey, I made the changes you requested.

As for your question
> While I'm reviewing, a question for you: would you like to publish it as soon as its ready, or would you prefer to wait until holidays/New Year's Eve celebrations are over?

I leave that up to you, as long as you ping me when it's online :-)

**Comment by ericwbailey** [comment]
Thanks for making the changes, @rianrietveld! I think we'll post after the start of the new year. 

Could you please [send us a message](https://www.a11yproject.com/contact/) to work out payment info and what info you'd like to use for your [author bio](https://www.a11yproject.com/authors/)?

**Review by ericwbailey — APPROVED** [review]

**Pull request merged**